### PR TITLE
[WIP] Add note about issues with Java 11 and Debian or RPM

### PR DESCRIPTION
**Pull request by karenzone**
I wanted to get a PR started so we can iterate.  This issue is related to install packages and Java 11, so I'm putting it in the Installation section for now.  We can move or duplicate as appropriate. Wording is also up for discussion.

### CODE REVIEW

**Comment by karenzone** [comment]
@jsvd  As discussed in team meeting.

File: docs/static/getting-started-with-logstash.asciidoc [hunk 1]
```diff
@@ -41,6 +41,9 @@ automatically detect your environment and install the correct startup method
 JAVA_HOME environment variable during package installation time, you may get an
 error message, and Logstash will be unable to start properly.
 
+IMPORTANT: Users have reported issues with Debian or RPM install packages and Java 11. 
+We are investigating.
```
Comment (jsvd) [inline docs/static/getting-started-with-logstash.asciidoc]: can you add a link to the related issue: https://github.com/elastic/logstash/issues/10593

**Comment by karenzone** [comment]
I updated the content with the GH link.  I changed the PR status to Ready For Review.  We can continue to iterate if necessary.

**Comment by jsvd** [comment]
Also merged to:

* 6d2447aaa - (6.7) Add note about issues with Java 11 and Debian or RPM (#10635) (40 seconds ago) <Karen Metts>
* cc27840a2 - (7.0) Add note about issues with Java 11 and Debian or RPM (#10635) (60 seconds ago) <Karen Metts>
* 9827ba8bb - (7.x) Add note about issues with Java 11 and Debian or RPM (#10635) (2 minutes ago) <Karen Metts>

**Pull request merged**